src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -1638,6 +1638,26 @@ public static Conversion ClassifyConversion(this SemanticModel? semanticModel, i
             var csModel = semanticModel as CSharpSemanticModel;
             return csModel?.GetInterceptorMethod(node, cancellationToken);
         }
+
+        /// <summary>
+        /// If 'node' cannot be intercepted syntactically, returns null.
```

Comment:
Do we have a test for the case of a `null` return because the location is syntactically considered to be not-interceptable?